<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeForm</class>
 <widget class="QDialog" name="WelcomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeLabelText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome to Setup</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Mouse">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Classical Mouse (Default)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Android_app">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Andriod App</string>
   </property>
  </widget>
  <widget class="QLabel" name="HintTextConnectivity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the connectivity mode:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NextStepPushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QListWidget" name="OnlineConnectionInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>256</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="OnlineHint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To connect with VNC Remote, Please connect your device to the wireless netwrok and then use the information below for VNC&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="OnlineHint_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Interfaces Address:</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Languages/build-MyMosqu-Desktop-Debug/Untitled-1.png);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="VNC_Remote">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>VNC Remote</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Wireless Network</string>
   </property>
   <widget class="QPushButton" name="Add_Network_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Network ESSID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Network_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_Field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Erease_all_Button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Erease all</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="on_screen_keyboard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Keyboard OnScreen</string>
   </property>
  </widget>
  <zorder>line</zorder>
  <zorder>WelcomeLabelText</zorder>
  <zorder>Mouse</zorder>
  <zorder>Android_app</zorder>
  <zorder>HintTextConnectivity</zorder>
  <zorder>OnlineConnectionInfo</zorder>
  <zorder>line_2</zorder>
  <zorder>OnlineHint</zorder>
  <zorder>OnlineHint_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>NextStepPushButton</zorder>
  <zorder>VNC_Remote</zorder>
  <zorder>groupBox</zorder>
  <zorder>on_screen_keyboard</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
